python/lsst/ts/mtmount/mock/controller.py

```diff
@@ -907,6 +940,71 @@ async def handle_command(self, command):
             # that `Controller.monitor_command` is awaiting.
             asyncio.create_task(self.monitor_command(command=command, task=task))
 
+    def run_commands_in_parallel(self, *commands):
+        """Run a set of commands in parallel.
+
+        Used to handle meta-commands that run a set of sub-commands
+        that can run at the same time.
+
+        Parameters
+        ----------
+        *commands : list [ `Command` ]
+            Commands to process.
+        """
+        if not self.command_server.connected:
+            raise RuntimeError("Command server not connected")
+
+        timeouts = []
+        tasks = []
+        for command in commands:
+
+            command_func = self.command_dict.get(command.command_code)
+            if command_func is None:
+                raise RuntimeError(f"command {command} is not supported")
+
+            timeout_task = command_func(command)
```

Comment:
Maybe wrap command_func line in try: catch Exception as e, similar to handle_command? I am not sure about that, but if running command signals an exception, you shall log it and reject the command, even if you are running two at the time - otherwise Python will log exception to stderr, where it will be hard to find.

Reply:
There is no need to do this because handle_command takes care of catching any exceptions. run_commands_in_parallel is used by some of the command handlers that handle_command calls.